lib/cqm/converter/bonnie_measure.rb

```diff
@@ -147,6 +140,30 @@ def self.to_cqm(bonnie_measure)
       cqm_measure
     end
 
+    def self.convert_observations(bonnie_measure, main_cql_library)
+      observations = []
+      bonnie_measure&.observations&.each_with_index do |bonnie_observation, observation_index|
+        # if this happens to be a multiple observation measure _this is unlikely_ we need to make a key to grab the proper hqmf_id
+        observation_population_key = observation_index.positive? ? "OBSERV_#{observation_index - 1}" : 'OBSERV'
```

Comment:
Is there a test for this? I can't find `OBSERV_#{number}` in the repo. It looks like this is assuming a multiple observation measure has keys of "OBSERV", and also "OBSERV_0", is that correct?

Reply:
We don't have any measures with multiple observations to test this code with. Multiple observation measures are possible but haven't shown themselves in the real world yet.

Reply:
I have no clue why i did the `- 1` gonna remove that.

Reply:
We won't be testing the support of conversion of multiple observation measures. This is because cqm-converters is a temporary project for our conversion and we already know we do not have to deal with multiple observation measures. A mongo query to check for this is:
```javascript
db.getCollection('cql_measures').find({ observations: {$exists: true}, $where: 'this.observations.length > 1'})
```